### ISSUE TITLE
Hotkey to toggle only the decks for smaller screens

### ISSUE THREAD

**Issue by CJNiiva** [HearthSim/Hearthstone-Deck-Tracker#4594]
I can toggle everything except the biggest UI element, my steamdeck screen (and ipad) is too small, hides the emd turn button, as of right now I have to toggle everything:(

**Comment by beheh** [comment]
So just to be clear, you're looking for a new hotkey action to toggle the own deck?

**Comment by CJNiiva** [comment]
No, to toggle the decklist for both you and opponent, its too large on a small screen

Sendt fra Outlook for iOS<https://aka.ms/o0ukef>
________________________________
